test/pystorm/test_component.py

```diff
@@ -272,6 +238,38 @@ def test_read_tuple(self):
             tup = component.read_tuple()
             self.assertEqual(output, tup)
 
+    def test_read_tuple_named_fields(self):
+        # This is only valid for bolts, so we only need to test with task IDs
+        # and Tuples
```

Comment:
That comment is copy-pasta from `test_read_tuple`, but you raise a good point.  The reason it's in `Component` is that when I got rid of the old `streamparse.ipc` module, I moved everything that was a function in there to be a method of the `Component` class.  It wasn't until later that I realized that you would never read a tuple in from a `Spout`.  I should probably clean that up.